Dockerfile

```diff
@@ -1,29 +1,47 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
 
 COPY . /tmp/workspace_deluxe
-RUN pip install configobj && \
-    cd /tmp && \
+WORKDIR /tmp
+RUN apt-get update -y && \
+    apt-get install -y ant git ca-certificates python3-sphinx && \
     git clone https://github.com/kbase/jars && \
     cd workspace_deluxe && \
     make docker_deps
 
-FROM kbase/kb_jre
+# updated/slimmed down version of what's in kbase/kb_jre
+FROM ubuntu:18.04
 
 # These ARGs values are passed in via the docker build command
 ARG BUILD_DATE
 ARG VCS_REF
 ARG BRANCH=develop
 
-COPY --from=build /tmp/workspace_deluxe/deployment/ /kb/deployment/
-
-RUN /usr/bin/tomcat8-instance-create /kb/deployment/services/workspace/tomcat && \
-    mv /kb/deployment/services/workspace/WorkspaceService.war /kb/deployment/services/workspace/tomcat/webapps/ROOT.war && \
-    rm -rf /kb/deployment/services/workspace/tomcat/webapps/ROOT
-
 # Must set catalina_base to match location of tomcat8-instance-create dir
 # before calling /usr/share/tomcat8/bin/catalina.sh
 ENV CATALINA_BASE /kb/deployment/services/workspace/tomcat
 ENV KB_DEPLOYMENT_CONFIG /kb/deployment/conf/deployment.cfg
+ENV DOCKERIZE_VERSION linux-amd64-v0.6.1
+ENV TOMCAT_VERSION tomcat8
+USER root
+
+RUN mkdir -p /var/lib/apt/lists/partial && \
+    apt-get update -y && \
+    apt-get install --no-install-recommends -y ca-certificates ${TOMCAT_VERSION}-user libservlet3.1-java wget && \
+    update-ca-certificates && \
+    apt-get clean && \
+    useradd -c "KBase user" -rd /kb/deployment/ -u 998 -s /bin/bash kbase && \
+    mkdir -p /kb/deployment/bin && \
```

Comment:
Looks like these two lines are missing, was that intentional?
```
    mkdir -p /kb/deployment/jettybase/logs/ && \
    touch /kb/deployment/jettybase/logs/request.log && \
```

Reply:
Yes -- those folders aren't used as tomcat does the server work on this image. Other KBase java containers -- e.g. the auth2 image -- use jetty as a server so this sets up the appropriate directories for them.

Reply:
:+1: